Packages/com.unity.inputsystem/Documentation~/HowDoI.md

```diff
@@ -1,595 +1,47 @@
 # How do I…?
 
-Devices:
+A collection of frequently asked questions, and where to find their answers in the documentation.
 
-* […check if the space key has been pressed this frame?](#check-if-the-space-key-has-been-pressed-this-frame)
-* […find all connected gamepads?](#find-all-connected-gamepads)
-* […find the gamepad that the player is currently using?](#find-the-gamepad-that-the-player-is-currently-using)
-* […know when a new device has been plugged in?](#know-when-a-new-device-has-been-plugged-in)
+How do I...?
 
-Actions:
+- [check if a specific key or button was pressed this frame?](../api/UnityEngine.InputSystem.Controls.ButtonControl.html#UnityEngine_InputSystem_Controls_ButtonControl_wasPressedThisFrame)
+  
+- [check if any key or button was pressed](../api/UnityEngine.InputSystem.InputSystem.html#UnityEngine_InputSystem_InputSystem_onAnyButtonPress)
 
-* […create a simple Fire-type Action?](#create-a-simple-fire-type-action)
-* […require a button to be held for 0.4 seconds before triggering an Action?](#require-a-button-to-be-held-for-04-seconds-before-triggering-an-action)
-* […use a "positive" and a "negative" button to drive an axis?](#use-a-positive-and-a-negative-button-to-drive-an-axis)
-* […create a UI to rebind input in my game?](#create-a-ui-to-rebind-input-in-my-game)
```

Comment:
create-a-ui-to-rebind-input-in-my-game is removed here but referenced from the joystick.md and denbugging.md files

Reply:
fixed